database/migrations/2021_12_04_181200_refactor_models.php

```diff
@@ -319,7 +319,7 @@ public function down()
 		$this->createUserAlbumTableDown();
 		$this->createPhotoTableDown();
 		$this->createSymLinkTableDown();
-		$this->createWebAuthnTableDown();
+		// $this->createWebAuthnTableDown();
```

Comment:
See above.

Reply:
fixed [faf44a2](https://github.com/LycheeOrg/Lychee/pull/1469/commits/faf44a2f63086be3902560abe44d2c11a8daf791)

Reply:
I am not sure if I like the fix of https://github.com/LycheeOrg/Lychee/pull/1469/commits/faf44a2f63086be3902560abe44d2c11a8daf791. Yes, it improves things in the sense that it removes code which has been commented out. But the whole change still touches old migrations which I would definitely not do. I would simply keep the old migrations as is, even if these migrations do things which turn out to be unnecessary later.

Also see  https://github.com/LycheeOrg/Lychee/pull/1469#discussion_r985269381.